src/middlewares/logRequests.ts

```diff
@@ -0,0 +1,40 @@
+import { NextFunction, Response, Request } from "express";
+
+import Logger from "../config/Logger";
+
+interface RequestLog {
+  body?: string;
+  query: string;
+  headers?: string;
+  baseUrl: string;
+  originalUrl: string;
+  params?: string;
+  hostname: string;
+  method: string;
+}
+
+const logRequest = (req: Request, res: Response, next: NextFunction) => {
+  const request: RequestLog = {
+    baseUrl: req.baseUrl,
+    originalUrl: req.originalUrl,
+    hostname: req.hostname,
+    method: req.method,
+    params: JSON.stringify(req.params),
+    query: JSON.stringify(req.query),
+  };
+  // if request has not big body add body
+  // and hide password field in it
+  if (req.body) {
+    if (JSON.stringify(req.body).length < 200) {
+      request.body = req.body?.password
```

Comment:
try to find native windston functionality to do same but properly

Reply:
I couldn't find any interesting option to change it with winston. I can only move such logic to Loger.ts, but it will be more horrible there.